src/app/models/hyp3.model.ts

```diff
@@ -24,9 +24,11 @@ export interface Hyp3JobSubmission {
 }
 
 export interface Hyp3User {
+  application_status: string;
```

Comment:
I would recommend changing the application status to be an enum with all the possible values.